src/map/latent_effect.h

```diff
@@ -36,6 +36,7 @@ enum LATENT
     LATENT_HP_OVER_VISIBLE_GEAR     = 46, //hp more than or equal to %, calculated using HP bonuses from visible gear only 
     LATENT_MP_UNDER_PERCENT         = 4,  //mp less than or equal to % - PARAM: MP PERCENT
     LATENT_MP_UNDER                 = 5,  //mp less than # - PARAM: MP #
+    LATENT_MP_OVER                  = 49, //mp greater than # - PARAM: MP #

```

Comment:
https://github.com/DarkstarProject/darkstar/pull/1994/files#diff-74b274188446a27a1f8589567c712eefR49
looks like 49 is already LATENT_NO_FOOD_ACTIVE..I wonder who's bright idea it was to try and organize these according to their sense of what looked nice..